README.md

```diff
@@ -18,6 +18,14 @@ Each station has a slightly different UI, allowing for different inputs, tools,
 
 It is important to remember that Wood Crafting is a system. This means that you can make your own module that houses your recipes and crafting stations so you won't have to modify Wood Crafting.
 
+## Getting started
+You're probably wondering how to actually craft an item. Here's how you get started.
+
+1. Whip out your pickaxe (No, a shovel won't work)
```

Comment:
Humor in readmes/tutorials can be dangerous, even if that's kinda sad. Soooo easy to misunderstand when the potential readerbase spans so many cultures and levels of expertise.